README.md

```diff
@@ -226,7 +226,8 @@ A sample configuration file with all options is available [here](https://github.
     * `"allow-null-check"` allows `==` and `!=` when comparing to `null`.
 * `typedef` enforces type definitions to exist. Rule options:
     * `"call-signature"` checks return type of functions.
-    * `"parameter"` checks type specifier of function parameters.
+    * `"parameter"` checks type specifier of function parameters for non-arrow functions.
+    * `"parameter-arrow"` checks type specifier of function parameters for arrow functions.
```

Comment:
Making this `arrow-func-parameter` would lead to nicer error messages. Any thoughts on naming @adidahiya and @tomduncalf?